v1.0/Specification.md

```diff
@@ -332,6 +332,7 @@ Table with terms used in the specification.
 | Term | Description |
 | --- | --- |
 | **Archival Information Package (AIP)** | *From OAIS:* An Information Package, consisting of the Content Information and the associated Preservation Description Information (PDI), which is preserved within an OAIS. |
+| **Artefact** | An artefact is a data object in a spreadsheet which may be considered insignificant for preservation and future reuse of the spreadsheet, but removing the object may not be considered neither necessary nor worthwhile. Therefore, the data object may remain in the spreadsheet and considered unnecessary for rendering or future migration. |
```

Comment:
So I wonder about the terminology here. You use the words "considered insignificant" in the description. This feels as though it's bumping up to the "significant properties" definition. Question, is an artefact in fact an insignificant property?

Reply:
I agree, we should use the term "significant property" in this definition.